cdap-support-bundle/src/main/java/io/cdap/cdap/task/SupportBundleTask.java

```diff
@@ -14,15 +14,16 @@
  * the License.
  */
 
-package io.cdap.cdap.support.task;
+package io.cdap.cdap.task;
 
 import io.cdap.cdap.common.NotFoundException;
+import io.kubernetes.client.openapi.ApiException;
 import org.json.JSONException;
 
 import java.io.IOException;
 
 /** Establishes an interface for support bundle task */
 public interface SupportBundleTask {
   /** Collect Log or pipeline info */
-  void collect() throws IOException, NotFoundException, JSONException;
+  void collect() throws IOException, NotFoundException, JSONException, ApiException;
```

Comment:
What does the `collect` method do? It may be good to have the `collect` interface method be defined as returning some data so some upstream caller can decide what to do with that data after-the-fact.